core/crypto/nike/hybrid/no_nobs_csidh.go

```diff
@@ -9,8 +8,8 @@ import (
 	"github.com/katzenpost/katzenpost/core/crypto/rand"
 )
 
-var CTIDHX25519 nike.Scheme = &scheme{
-	name:   "CTIDH-X25519",
-	first:  nil,
-	second: ecdh.NewEcdhNike(rand.Reader),
+var NOBS_CSIDHX25519 nike.Scheme = &scheme{
+	name:   "NOBS_CSIDH-X25519",
+	first:  ecdh.NewEcdhNike(rand.Reader),
+	second: nil,
```

Comment:
Since this isn't using CSIDH on `ppc64le`, what's the point of defining it?
Ie we should have static test vectors for this, and those would then fail on `ppc6le`.
Seems better to have it unavailable on `ppc64le` altogether?

Reply:
This is a convenience to allow us to compile our mix server and dirauth server on that ppc64le hardware. These are not even used by the mix server or the dirauth server... but the build would fail.

Reply:
I'm fine with removing it but then mixserver and dirauth won't build on ppc64le.

Reply:
Why is `NOBS_CSIDHX25519` required to build `mixserver` and `dirauth` on `ppc64le` when the implementation isn't required to work, or called?

Reply:
Seems to me it would be more appropriate to have `geo.go` parametrically compiled then, to only reference libraries available on the target platform?

Reply:
Seems to me it would be more appropriate to have `geo.go` parametrically compiled then, to only reference libraries available on the target platform?